spring-integration-core/src/test/java/org/springframework/integration/channel/QueueChannelTests.java

```diff
@@ -248,6 +249,7 @@ public void testPurge() {
 	public final TemporaryFolder tempFolder = new TemporaryFolder();
 
 	@Test
+	@Ignore
```

Comment:
Why is this ignored? It sometimes works for me; sometimes fails.

And, there is a bad `latch2.await();` (no timeout) (and latch3)

If this is a known reactor problem, add a comment pointing to the reactor issue after `@Ignore`